CovertInfiltration/Src/CovertInfiltration/Classes/X2TemplateModifier.uc

```diff
@@ -0,0 +1,133 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  statusNone
+//  PURPOSE: Modifys Templates in order to make infinite items single build instead
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+class X2TemplateModifier extends Object config (SingleBuildItems);
+
+struct SingleBuildItem
+{
+	var name SchematicName;
+	var int TradingPostValue;
+	
+	var array<StrategyCost> Costs;
```

Comment:
Instead of diff -> value array, add the difficulty as another field in this struct. That way it will be 1 difficulty per line in config files - easier to read and change